src/views/sidebar/new-modal.tsx

```diff
@@ -27,8 +27,27 @@ export const NewModal: FC<ModalProps> = ({ folder, onClose }) => {
 	const [label, setLabel] = useState<string>(
 		t('folder_panel.modal.new.input.name', 'Enter Folder Name')
 	);
-
-	const showWarning = useMemo(() => includes(translatedSystemFolders(), inputValue), [inputValue]);
+	const [errorMsg, setErrorMsg] = useState<string>(
+		t('folder.modal.edit.rename_warning', 'You cannot rename a folder as a system one.')
+	);
+	const showWarning = useMemo(() => {
+		if (includes(translatedSystemFolders(), inputValue)) {
+			setErrorMsg(
+				t('folder.modal.edit.rename_warning', 'You cannot rename a folder as a system one.')
+			);
+			return true;
+		}
+		if (inputValue && inputValue.includes('/')) {
```

Comment:
Are you sure this is the only character which is not allowed?

Reply:
Further research brought me here:

Name of folder; max length 128; whitespace is trimmed by server; Cannot contain ':', '"', '/', or any character below 0x20